packages/components/psammead-section-label/src/titles.jsx

```diff
@@ -75,7 +75,7 @@ const Title = styled.span`
   ${({ script }) => script && getDoublePica(script)};
   ${({ service }) => getSansRegular(service)}
   color: ${C_EBON};
-  background-color: ${C_WHITE};
+  background-color: ${C_GHOST};
```

Comment:
I mentioned this in a comment, but it has collapsed.

It'd be good to pass in the background colour as a prop e.g. backgroundColour, so we can use this section label on multiple different pages (I know in the near future we'll have a dark mode for some pages, so adding this as a prop would help for that, too.)